### PR TITLE
AG-12294 Update icons

### PR DESCRIPTION
**Pull request by lsjroberts**
https://ag-grid.atlassian.net/browse/AG-12294
https://ag-grid.atlassian.net/browse/AG-12144

### CODE REVIEW

**Review by jacobp100 — APPROVED** [review]

File: packages/ag-charts-types/src/chart/toolbarOptions.ts [hunk 1]
```diff
@@ -45,28 +45,65 @@ export interface AgToolbarButton {
     value: any;
 }
 
-export type AgIconName = IconNameAnnotation | IconNameZoom;
```
Comment (alantreadway) [inline packages/ag-charts-types/src/chart/toolbarOptions.ts:48]: These look like breaking changes - were these types used in publicly documented options?

Comment (alantreadway) [reply packages/ag-charts-types/src/chart/toolbarOptions.ts:48]: Discussed offline, there are two cases that we need to deprecate as they are not in the new icon set:
- `zoom-in-alt`
- `zoom-out-alt`

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**